src/Conversion/ONNXToTOSA/Tensor/Resize.cpp

```diff
@@ -0,0 +1,294 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===---------------- Resize.cpp - Resize Op-------------------------------===//
+//
+// Copyright (c) 2022 Advanced Micro Devices, Inc.
+//
+// =============================================================================
+//
+// This file lowers the ONNX Resize operator to TOSA dialect.
+//
+//===----------------------------------------------------------------------===//
+
+#include "mlir/Dialect/Tosa/IR/TosaOps.h"
+#include "src/Conversion/ONNXToTOSA/DialectBuilder.hpp"
+#include "src/Conversion/ONNXToTOSA/ONNXToTOSACommon.hpp"
+#include "src/Conversion/ONNXToTOSA/ONNXToTOSALegalizeUtils.hpp"
+#include <cstdint>
+#include <numeric>
+
+using namespace mlir;
+
+namespace onnx_mlir {
+
+namespace {
+
+std::tuple<int64_t, int64_t> floatToFraction(float number) {
```

Comment:
Tuple are smelly (https://arne-mertz.de/2017/03/smelly-pair-tuple/), maybe you can make a `struct`? Something like:
```suggestion
struct fractionNumber{
int64_t numerator;
int64_t denominator;
};
```